tests/test-mul-mat2.c

```diff
@@ -255,8 +253,8 @@ void mul_mat_gq_1(
                     s1[b + 1] = d1*(1 << b);
                 }
 
-                m0[0] = -1ULL;
-                m1[0] = -1ULL;
+                m0[0] = 0-1ULL;
+                m1[0] = 0-1ULL;
```

Comment:
What warning does this fix?

Reply:
"unary minus applied to unsigned value, result is still unsigned"